dime-coding-standards/checklists/tex/peer_code_review_guidance_note.tex

```diff
@@ -0,0 +1,127 @@
+\fancyhf{}
+\lfoot{\includegraphics[height=1cm,keepaspectratio]{../../img/i2i}}
+\cfoot{\includegraphics[height=1cm,keepaspectratio]{../../img/wb}}
+\rfoot{\includegraphics[height=1cm,keepaspectratio]{../../img/analytics}}
+
+% Put checkbox to the left of the text
+\def\LayoutCheckField#1#2{% label, field
+	#2 #1%
+}
+
+% Line spacing
+\onehalfspacing
+
+% Define DIME Analytics visual identity colors
+\definecolor{fontcolor}{HTML}{7A0569}
+
+\titleformat{\section}%
+{\Large\rmfamily\bf\color{fontcolor}}% format applied to label+text
+{\llap{\colorbox{fontcolor}{\parbox{1.5cm}{\hfill\huge\color{fontcolor}\thesection}}}}% label
+{2pt}% horizontal separation between label and title body
+{}% before the title body
+[]% after the title body
+
+\titleformat{\subsection}%
+{\large\rmfamily\color{fontcolor}}% format applied to label+text
+{}% label
+{1.5pt}% horizontal separation between label and title body
+{}% before the title body
+[]% after the title body
+
+\newcommand{\dimeCheckBox}[1]{\CheckBox[height=0.01cm, width=0.4cm, bordercolor=gray]{#1}}
+\newcommand{\dimeTextField}[3]{\TextField[name=#1, height=0.3cm, width=#2, bordercolor=gray]{#3}}
+
+
+\newcommand{\titleBox}[1]{
+	\begin{tcolorbox}
+		[colframe = fontcolor,
+		colback = fontcolor,
+		sharp corners,
+		halign = flush center,
+		valign = center,
+		height = 0.3\textwidth,
+		after skip = 1cm]
+		#1
+	\end{tcolorbox}
+}
+
+
+
+\begin{document}
+	\begin{fullwidth}
+
+\titleBox{
+	\textcolor{white}{\LARGE{\textbf{DIME Analytics \\ Quarterly Peer Code Review Guidance Note}} \\
+	\Large\textbf{{v1.0 - Last updated November 18, 2022}}}
+}
+
+	\section*{Overview}
+ Every quarter, DIME Analytics organizes the \textbf{Peer Code Review}, which is a real-time code and data-quality assurance process. This is a structured opportunity for participants to exchange, run, and provide feedback on each other’s code. Apart from increasing reproducibility, transparency and adherence to best practices, it is a great learning opportunity - code writers get to look at their work from a different perspective, and reviewers are exposed to different styles and practices. 
+\hfill \break
+\begin{flushleft} The peer code review is designed for scripts that are modular enough to be understood on their own (a reviewer can understand the script without having to refer to other project code files) and are short enough for a reviewer to read through in approximately half a day (depending on complexity). We recommend submitting recently-completed tasks, to allow corrections to be made in real time.
+
+\section*{How it works}
+
+	\begin{enumerate}
+	    
+	  
+		\setlength\itemsep{-0.1em}
+		\item Teams \href{https://survey.wb.surveycto.com/collect/code_review_sign_up?caseid=}{sign-up} to participate in the code review each quarter. All DIME Research Assistants working on code are expected to participate, and any Bank staff or consultant who would like to get feedback on their code is welcome to participate.
```

Comment:
We should think about this link. as it will be public we should make sure sign-up is off except during the designated sign-up periods, or make sure that the dates for the next round are clear. like it wouldn't be a problem if someone signs up from another unit now if they know the round they are signing up for doesn't take place until February. maybe ideal is that in the periods in-between sign-ups there's just a message that says very clearly that the next round won't be until X date and that sign-up will begin at Y date.